requirements.txt

```diff
@@ -1,4 +1,3 @@
-lxml==4.9.3

```

Comment:
why remove these packages?

Reply:
They are moved to projectpy.toml and are installed locally with the `-e .` line.